app/src/userProfileController.js

```diff
@@ -0,0 +1,51 @@
+(function () {
+    angular
+        .module('webshopModule')
+        .controller('userProfileCtrl', UserProfileController);
+    function UserProfileController(api, loginService, $scope) {
+        var session = loginService.getSession();
+
+        function getUser() {
+            api.getUserId(session.token)
+                .then(function (result) {
+                    api.getUser(session.token, result.data.id);
+                    return result;
+                })
+                .then(function (result) {
+                    $scope.user = result.data;
+                });
+        }
+
+        getUser();
+
+        var successfulEdit = function () {
+            $scope.successfulEdit = true;
+            $scope.unsuccessfulEdit = false;
+        };
+
+        var unsuccessfulEdit = function () {
+            $scope.unsuccessfulEdit = true;
```

Comment:
When set, the value of these variables is mutually exclusive. Please use the negated value of `successfulEdit` instead of `unsuccessfulEdit`.